client/my-sites/email/email-management/titan-manage-mailboxes/index.jsx

```diff
@@ -192,5 +242,6 @@ export default connect( ( state, ownProps ) => {
 			isFetchingSitePurchases( state ) || ! hasLoadedSitePurchasesFromServer( state ),
 		purchase: getEmailPurchaseByDomain( state, domain ),
 		selectedSite,
+		isMobileViewport: isMobile(),
```

Comment:
Why do we need to pass this `isMobileViewport` prop? Can't we just call `isMobile()` from inside the component (like we do in other places in Calypso)?

Reply:
I wanted to prevent the `isMobile()` function calls on every render. I realise that it would be called on every re-render even if I pass it down as a prop.